app/models/branch.rb

```diff
@@ -0,0 +1,11 @@
+class Branch < ApplicationRecord
+  validates :name, :opening_time, :closing_time, presence: true
+  validates :opening_time, comparison: { less_than: :closing_time }, if: :closing_time
```

Comment:
Instead of this
Add validation on closing time, it should be later than opening time.

Reply:
Done